docs/reference/docs/delete-by-query.asciidoc

```diff
@@ -72,7 +72,7 @@ If you'd like to count version conflicts rather than cause them to abort then
 set `conflicts=proceed` on the url or `"conflicts": "proceed"` in the request body.
 
 Back to the API format, you can limit `_delete_by_query` to a single type. This
-will only delete `tweet` documents from the `twitter` index:
+will only delete `_doc` documents from the `twitter` index:
```

Comment:
This should probably be:

```
s/`tweet` documents/tweets
```

and not refer to the actual underlying type?

Reply:
What about removing the whole paragraph ;) 

> you can limit `_delete_by_query` to a single type

Reply:
I think the paragraph should stay in some form, probably something like:

> This will delete tweets from the `twitter` index:

The rest of it indeed does not make sense as you suggest as there are not multiple types to speak of anymore, so clarifying that an API invocation only acts on a single type is unnecessary.